operator-framework/src/test/java/io/javaoperatorsdk/operator/CRDMappingInTestExtensionIT.java

```diff
@@ -28,7 +28,7 @@ public class CRDMappingInTestExtensionIT {
   LocallyRunOperatorExtension operator =
       LocallyRunOperatorExtension.builder()
           .withReconciler(new TestReconciler())
-          .withCRDMapping("tests.crd.example", "src/test/crd/test.crd")
+          .withAdditionalCRD("src/test/resources/crd/test.crd")
```

Comment:
This was explicitly not put in `resources` to check that the file could be accessed from a "random" part of the file system, not just from the classpath.

Reply:
I see, isn't it trivial?  I would rather stick with this, but no strong opinion, if you insist can change it back.